<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackupFormDialog</class>
 <widget class="QWidget" name="BackupFormDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="nameInput">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>31</y>
     <width>301</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Nom Sauvegarde</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="icon">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string>icon</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>79</y>
     <width>45</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Source</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sourceInput">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>79</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>175</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="sourceChoiceButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>79</y>
     <width>40</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="targetChoiceButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>119</y>
     <width>40</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="targetInput">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>119</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>175</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>119</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cible</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="backupButtonBox">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>315</y>
     <width>301</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_1">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>158</y>
     <width>302</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Fr√©quence</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QTimeEdit" name="timeEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>198</y>
     <width>302</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Note</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="noteInput"/>
    </item>
   </layout>
  </widget>
  <zorder>nameInput</zorder>
  <zorder>label_2</zorder>
  <zorder>sourceChoiceButton</zorder>
  <zorder>targetChoiceButton</zorder>
  <zorder>label_3</zorder>
  <zorder>backupButtonBox</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>icon</zorder>
  <zorder>targetInput</zorder>
  <zorder>sourceInput</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
